README.md

```diff
@@ -1,11 +1,49 @@
-# Drive
+<p align="center">
+  <a href="https://github.com/suitenumerique/drive">
+    <img alt="Drive Logo" src="/docs/assets/drive-logo-white-background.png" width="300" />
+  </a>
+</p>
 
-Drive is an application for managing files in a collaborative way
+<p align="center">
+Welcome to Drive! The open-source file management solution where your files become collaborative assets through seamless teamwork.
+</p>
 
-It is built on top of [Django Rest
-Framework](https://www.django-rest-framework.org/).
+<img alt="Drive Logo" src="/docs/assets/drive-logo-white-background.png" width="300" width="100%" align="center"/>
```

Comment:
why repeat the logo. It looks weird? https://github.com/suitenumerique/drive/blob/5932c31b390595fcaa3ae960ee2cf5ffff195173/README.md